frontend/src/layout/navigation-3000/Navigation.scss

```diff
@@ -71,7 +71,7 @@
         }
 
         li + li {
-            margin-top: 0.25rem;
+            margin-top: 1px;
```

Comment:
This could become a `space-y-px` class on `Navbar3000__content`

Reply:
Hmm, it could, but then we'd have more of a mix of utility classes and BEM. Gonna stick with BEM here